src/styles/main.scss

```diff
@@ -70,8 +98,11 @@ input[type="text"] {
 }
 
 #submit-btn {
+  width: 100px;
   padding: 5px;
   border: 1px solid #ccc;
   background-color: #fff;
   cursor: pointer;
+  border-radius: 3px;
+  box-shadow: 0 0 5px #ccc;
 }
```

Comment:
Hi @mohashyne, you did well with your leaderboard project 👍 . Please remember that you are to copy the design as closely as possible so please implement these changes :

1.  the `Add your score` should be a heading and it should be centralized,as it's a heading of a subsection
2.  The `Submit` button should be on the right, just as it is in the design given
3. The `recent scores` and the `Refresh` buttons shouldn't be together. the first is on the left while the other is on the right of the same subsection

Your design :
![leaderboard-image](https://github.com/mohashyne/leaderboard/assets/20537712/fb3e1aba-6d4e-4c16-9e3f-822e70c5a8e7)

Framework :

![image](https://github.com/mohashyne/leaderboard/assets/20537712/d192936c-5a91-471e-9708-e0de7dcd9de2)

Reply:
Requested changes applied